is_js/is_js.d.ts

```diff
@@ -1318,6 +1318,6 @@ interface Is extends IsStatic {
 
 declare var is: Is;
 
-declare module 'is' {
```

Comment:
http://arasatasaygin.github.io/is.js/
it has both. please keep old module definitions.